src/pages/api/proxy/[...path].ts

```diff
@@ -18,7 +26,7 @@ EventEmitter.defaultMaxListeners = Number(process.env.PROXY_DEFAULT_MAX_LISTENER
 // to the backend server, allowing for features like cookie handling and request body fixing, which are essential
 // for maintaining session state and ensuring correct request formatting while in a cross domain env.
 const apiProxy = createProxyMiddleware<NextApiRequest, NextApiResponse>({
-  target: process.env.NEXT_PUBLIC_AUTH_BASE_URL,
+  target: process.env.NEXT_PUBLIC_API_GATEWAY_URL,
```

Comment:
This env variable should be `API_GATEWAY_URL` instead of `NEXT_PUBLIC_API_GATEWAY_URL` so that we don't expose it to JS since it's only used server-side.